web/src/beta/features/Navbar/index.tsx

```diff
@@ -52,7 +82,43 @@ export default Navbar;
 
 const Wrapper = styled.div`
   display: flex;
+  flex-direction: row;
   justify-content: space-between;
-  height: 53px;
+  align-items: center;
+  padding: 8px 24px;
+  gap: 24px;
+  width: 1920px;
+  height: 51px;
   background: ${({ theme }) => theme.editorNavBar.bg};
+  box-shadow: 0px 0px 10px rgba(0, 0, 0, 0.5);
+`;
+const RightHeader = styled.div`
+  display: flex;
+  flex-direction: row;
+  align-items: flex-end;   
+  padding-right:24px
+  gap: 4px;
+  width: 301px;
+  height: 35px;
+`;
+
+const TabButton = styled.button<{ isSelected: boolean }>`
```

Comment:
Can you not use components/Button as a base for the TabButtons?

Reply:
I prefer to use this button because the Button has more styles than needed so i wanted to keep it simple rather than add  a lot of css modifications

Reply:
done ( use TabButton  , move it inside navBar, modify the design)